pkg/controller/job/job_controller_test.go

```diff
@@ -798,6 +800,17 @@ func checkCompletionIndexesInPods(t *testing.T, control *controller.FakePodContr
 	}
 }
 
+func checkPodHostname(t *testing.T, control *controller.FakePodControl, jobName string) {
+	t.Helper()
+	for _, p := range control.Templates {
+		ix := getCompletionIndex(p.Annotations)
+		expectedName := fmt.Sprintf("%s-%s", jobName, strconv.Itoa(ix))
+		if diff := cmp.Diff(p.Spec.Hostname, expectedName); diff != "" {
```

Comment:
use `cmp.Equal` for single values.
Also, maybe do this check inside `checkCompletionIndexesInPods`, which can be renamed to `checkIndexedJobPods`

Reply:
Done